### PR TITLE
Remove forced coloring of blog links

### LINKED ISSUE #3127: Blog MVP: Link text not being differentiated from body text
**Describe the bug**
The new blog pages look great but Alan caught that link text are not showing up as being differentiated from the body text.

**To Reproduce**
1. Go to https://foundation.mofostaging.net/en/blog/blogs-are-live-staging/
2. Hover on 'Mozilla'
3. See error

**Expected behavior**
Mozilla is a link and should be Dark Blue from our style guide or #0d10bf

**Screenshots**

Current:
image

Should be dark blue like this link here:
Screen Shot 2019-05-10 at 2 22 21 PM

### PR DESCRIPTION
**Pull request by alanmoo**
Since links are always inside `p` elements, we don’t need to explicitly set the color.

Closes #3127

### CODE REVIEW

**Comment by alanmoo** [comment]
How did you create that link? I couldn't find a way to add a link using the admin without it being wrapped in a paragraph tag.

**Comment by mmmavis** [comment]
@alanmoo guess we should update our fake data generator so it's more realistic 😂

**Comment by alanmoo** [comment]
See #2786 :)

File: network-api/networkapi/utility/faker/streamfield_provider.py [hunk 1]
```diff
@@ -32,7 +33,8 @@ def generate_paragraph_field():
         f'<ul>',
         ''.join([f'<li>{fake.word()}</li>' for i in range(10)]),
         f'</ul><br />',
-        f'<a href="{fake.url(schemes=["https"])}">This is a link to a fake url!</a>'
+        f'<a href="{fake.url(schemes=["https"])}">This is a link to a fake url!</a>',
+        f'</p>'
```
Comment (mmmavis) [inline network-api/networkapi/utility/faker/streamfield_provider.py]: missing a closing </p>

Comment (mmmavis) [inline network-api/networkapi/utility/faker/streamfield_provider.py]: missing a closing `</p>`

Comment (alanmoo) [reply network-api/networkapi/utility/faker/streamfield_provider.py]: Yeah, I was having trouble getting the generation working locally. Still working on this.

Comment (alanmoo) [reply network-api/networkapi/utility/faker/streamfield_provider.py]: Now it resembles the markup that gets generated by the richtext component.

**Comment by mmmavis** [comment]
Did a quick test. It seems like Wagtail only auto wraps non-block level elements in `<p>`.
image

**Comment by alanmoo** [comment]
I just destroyed and recreated the review app.

https://foundation-mofostaging-pr-3140.herokuapp.com/en/blog/role-suggest-defense/

There's no extra closing paragraph tag in there, just the one I added in the test fixture to replicate the wrapping Wagtail's rich text editor does on save.

**Pull request merged**